app/src/main/res/values/strings.xml

```diff
@@ -16,10 +16,16 @@
     <string name="close">Uždaryti</string>
 
     <string name="problem_search">Problemos paieška</string>
+    <string name="progress_dialog_loading">Kraunama</string>
 
     <string name="profile_hint_email">E-mail</string>
     <string name="profile_hint_name">Vardas</string>
     <string name="profile_telephone">Telefonas</string>
+    <string name="message_notsigned">Norint pasiekti profilio informacija prašome prisijungti</string>
+    <string name="title_notsigned">Vartotojas neprisijungęs</string>
+    <string name="problem_send">Pranešti apie problemą</string>
+    <string name="login_sign_in">Sign in with Google</string>
```

Comment:
Hm... Let's keep language consistency in this file. We can add `/values-en/strings.xml` with all english translations.

@RenatBuga should we keep app's interface all lithuanian. Right?